gm4_smelteries/data/gm4_smelteries/functions/process.mcfunction

```diff
@@ -0,0 +1,16 @@
+# smeltery processes
+# @s - smeltery
+# called by main
+
+# removal
+execute unless block ~ ~ ~ hopper run function gm4_smelteries:destroy
+
+# verify furnace location
+execute unless block ^ ^ ^1 furnace run function gm4_smelteries:find_furnace
+
+# if furnace is lit, start checking for doubable items
+execute if entity @s[tag=gm4_bf_has_furnace] if block ^ ^ ^1 furnace[lit=true] run function gm4_smelteries:verify_furnace
+execute unless block ^ ^ ^1 furnace[lit=true] run replaceitem entity @s armor.head gray_stained_glass
```

Comment:
This isn't a very efficient way to make sure the display block is reset properly after smelting, but I can't think of a better way. Any thoughts on how to improve this?